fdbclient/GrvProxyInterface.h

```diff
@@ -0,0 +1,64 @@
+
+/*
+ * GrvProxyInterface.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
```

Comment:
```suggestion
 * Copyright 2013-2020 Apple Inc. and the FoundationDB project authors
```
To reflect changes made in 2020.

Reply:
Done.